packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -1994,6 +1994,156 @@ class FavoritesTable extends React.Component {
+    - `onCheckChange` - (optional) Callback when user changes the checkbox on a row.
+
+```js
+import React from 'react';
+import { Table, TableHeader, TableBody, headerCol, treeRow } from '@patternfly/react-table';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Table/table';
+
+class TreeTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: [
+        {
+          repositories: 'Repositories one',
+          branches: 'Branch one',
+          pullRequests: 'Pull request one',
+          workspaces: 'Workplace one',
+          children: [
+            {
+              repositories: 'Repositories two',
+              branches: 'Branch two',
+              pullRequests: 'Pull request two',
+              workspaces: 'Workplace two',
+              children: [
+                {
+                  repositories: 'Repositories three',
+                  branches: 'Branch three',
+                  pullRequests: 'Pull request three',
+                  workspaces: 'Workplace three',
+                }
+              ]
+            },
+            {
+              repositories: 'Repositories four',
+              branches: 'Branch four',
+              pullRequests: 'Pull request four',
+              workspaces: 'Workplace four',
+            },
+            {
+              repositories: 'Repositories five',
+              branches: 'Branch five',
+              pullRequests: 'Pull request five',
+              workspaces: 'Workplace five',
+            }
+          ]
+        },
+        {
+          repositories: 'Repositories six',
+          branches: 'Branch six',
+          pullRequests: 'Pull request six',
+          workspaces: 'Workplace six',
+          children: [
+            {
+              repositories: 'Repositories seven',
+              branches: 'Branch seven',
+              pullRequests: 'Pull request seven',
+              workspaces: 'Workplace seven'
+            }
+          ]
+        },
+        {
+          repositories: 'Repositories eight',
+          branches: 'Branch eight',
+          pullRequests: 'Pull request eight',
+          workspaces: 'Workplace eight'
+        }
+      ],
+      expandedRows: ['Repositories one', 'Repositories six'],
+      checkedRows: []
+    };
+    
+    this.buildRows = ([x, ...xs], level, posinset, isHidden = false) => {
+      if (x) {
+        const isExpanded = this.state.expandedRows.includes(x.repositories);
+        const isChecked = this.state.checkedRows.includes(x.repositories);
+        return [
+          {
+            cells: [x.repositories, x.branches, x.pullRequests, x.workspaces],
+            props: {
+              isExpanded: isExpanded,
+              isHidden,
+              level: level,
+              posinset: posinset,
+              setsize: x.children ? x.children.length : 0,
```

Comment:
can this be camel cased everywhere? `setSize`

Reply:
it could, but the aria attribute is not camel cased (i.e. `aria-posinset`, `aria-setsize`), so I figured it was less confusing to format it the same way. I could also just use the actual aria attribute name, but number of quotes i'd need to wrap the attribute name annoyed me.

What do you vote? @tlabaj @kmcfaul?

Reply:
I think using `'aria-posinest'` and `'aria-setsize'` with quotes would be the clearest. Or at least having "aria" in the property name to be clear what the prop is intended for ( i.e. `ariaPosInset` / `ariaSetSize` )